MovieQuiz-ios-project_sprint_3_start-3/MovieQuiz/Presentation/AlertPresenter.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  AlertPresenter.swift
+//  MovieQuiz
+//
+//  Created by Danzl Wa on 28.11.2022.
+//
+
+import Foundation
+import UIKit
+
+class AlertPresenter {
+    
+    let alertModel: AlertModel
+    weak var viewController: UIViewController?
+    
+    init(alertModel: AlertModel, viewController: UIViewController) {
+        self.alertModel = alertModel
+        self.viewController = viewController
+    }
+    
+    func showResultsAlert() {
+        let alert = UIAlertController(title: alertModel.title, message: alertModel.message, preferredStyle: .alert)
+        let action = UIAlertAction(title: alertModel.buttonText, style: .default) { _ in
+            self.alertModel.completion()
```

Comment:
здесь может произойти утечка памяти, лучше захватить слабо ссылку. 
```suggestion
        let action = UIAlertAction(title: alertModel.buttonText, style: .default) { [weak self] _ in
            self?.alertModel.completion()
```